.cirrus.yml

```diff
@@ -1,15 +1,27 @@
+env:
+  BUILD_DIR: Dist
+
 task:
   name: Test FreeBSD
   freebsd_instance:
     matrix:
       image_family: freebsd-12-3
       image_family: freebsd-13-0
   install_script:
-    - pkg install -y libpcap bash gsed gmake tcpreplay
+    - pkg install -y libpcap bash gsed git gmake tcpreplay cmake
+  clone_script: |
+    if [ -z "$CIRRUS_PR" ]; then
+      git clone --recursive --branch=$CIRRUS_BRANCH https://x-access-token:${CIRRUS_REPO_CLONE_TOKEN}@github.com/${CIRRUS_REPO_FULL_NAME}.git $CIRRUS_WORKING_DIR
+      git reset --hard $CIRRUS_CHANGE_IN_REPO
+    else
+      git clone --recursive https://x-access-token:${CIRRUS_REPO_CLONE_TOKEN}@github.com/${CIRRUS_REPO_FULL_NAME}.git $CIRRUS_WORKING_DIR
+      git fetch origin pull/$CIRRUS_PR/head:pull/$CIRRUS_PR
+      git reset --hard $CIRRUS_CHANGE_IN_REPO
+    fi
```

Comment:
I'm not sure I understand this change

Reply:
If we get the tag from git we need to clone the repo with all the reference to get the proper tags